src/index.js

```diff
@@ -33,8 +33,8 @@ export default {
   List: require('./lists/list'),
   ListDivider: require('./lists/list-divider'),
   ListItem: require('./lists/list-item'),
-  Menu: require('./menu/menu'),
-  MenuItem: require('./menu/menu-item'),
+  Menu: require('./menus/menu'),
```

Comment:
waiting that long will piss people off too! let's just add notes to Changelog about this breaking change and ways to use the old menu.